common/content/finder.js

```diff
@@ -274,6 +290,18 @@ var Finder = Module("finder", {
             function () { finder.clear(); },
             { argCount: "0" });
     },
+    events: function() {
+        if (config.name == 'Vimperator') {
+            // Same thing as the note above for not showing the findbar on failed results
+            // through the command line, but for compatibility with FindBar Tweak.
+            events.addSessionListener(window, "WillOpenFindBar", function (event) {
```

Comment:
will findbar tweak respect the options above? I would prefer to keep other addon-specific fixed out of the codebase if at all possible.